src/Files.App/Views/Properties/SecurityPage.xaml

```diff
@@ -140,6 +144,12 @@
 					</ListView>
 				</ScrollViewer>
 
+				<TextBlock
+					x:Name="NoAcesTextBlock"
```

Comment:
Should this be "Access"?

Reply:
No, No ACE TextBlock. ACE is AccessControlEntry. Should I use it instead?

Reply:
Whichever is clearer in the code